run-test-local.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -euo pipefail
+
+export GITHUB_WORKSPACE=$(pwd)
+
+docker pull dev.registry.tanzu.vmware.com/app-accelerator/acc-engine:latest
```

Comment:
this is not publicly accessible, we should not commit this to the repo

Reply:
Merged your changes from other PR and put the docker commands in separate script. I hope that's okay. If you are objecting to the mere presence of the string`dev.registry.tanzu.vmware.com/app-accelerator/acc-engine:latest` anywhere in the repo... I will point out that this string can also be found inside the .github workflow definitions.